### PR TITLE
nodejs-fr: bootstrap about/governance page

### PR DESCRIPTION
**Pull request by Tiriel**
Hey there!

Another first draft, hopefully with less typos than the first.
I'm open to revisions as usual!

Cheers

### CODE REVIEW

File: locale/fr/about/governance.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+la gouvernance de haut-niveau du projet.
+
+Le TSC a toute autorité sur ce projet, y compris:
+
+* la Direction Technique
+* La gouvernance et la gestion du projet (y compris cette politique)
+* La politique de contribution
+* la gestion des dépôts GitHub
+* Les guides de Conduite
+* La maintenance de la liste des collaborateurs
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: Unifier les majuscules dans la liste au début de chaque ligne.

et

> la liste des Collaborateurs

File: locale/fr/about/governance.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+d'évoluer dans le temps en rapport avec les besoins du projet.
+
+Vous pouvez trouver la liste des membres actuels du TSC sur le projet
+[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
+
+## Collaborateurs
+
+Le dépôt GitHub [nodejs/node](https://github.com/nodejs/node) est 
+maintenu par le TSC et un groupe de Collaborateurs additionnels
+Qui sont ajoutés par le TSC de manière continue.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > **q**ui

pas besoin de majuscule ici.

File: locale/fr/about/governance.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+Les personnes proposant des contributions significatives sont faits
+Collaborateurs et se voient accordé les droits d'écriture sur le projet.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > se voient accord**er**

File: locale/fr/about/governance.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+obtenu les droits de commit, ouvrez un ticket ou contactez un membre du TSC
+directement et votre demande sera examinée lors de la prochaine réunion.
+
+Les modifications de contenu sur le dépôt nodejs/node sont validées de
+manière collaborative. N'importe qui possédant un compte GitHub peut
+proposer une modification par pull request et elle sera examinée par les
+Collaborateurs du projet. Toutes les pull request doivent être relues et acceptées
+par un Collaborateur ayant une expertise suffisante et qui soit capable
+d'assumer la résponsabilité du changement effectué. Dans le cas d'une pull
+request proposée par un Collaborateur, un autre Collaborator est requis
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un autre Collaborateur

File: locale/fr/about/governance.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+Les Collaborateurs peuvent choisir de faire remonter au TSC pour
+discussion des modifications significatives ou sujettes à controverse,
+ou des modifications n'ayant pas obtenu de consensus, en leur assignant
+l'étiquette ***tsc-agenda*** sur une pull request ou un ticket. Le
+TSC servira alors d'arbitre final lorsque requis.
+
+Pour la liste des Collaborateurs actuels, voir le projet 
+[README.md](https://github.com/nodejs/node/blob/master/README.md#current-project-team-members).
+
+Un guide des Collaborateurs et proposé sur
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > est proposé sur

File: locale/fr/about/governance.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+efficace.
+
+Il n'y a aucun prérequis ou qualifications nécessaires au delà de
+ces règles pour siéger au TSC.
+
+Le TSC peut ajouter des membres additionnels au TSC par une motion
+standard du TSC.
+
+Un membre du TSC peut être retiré du TSC par démission volontaire, ou pas
+par une motion standard du TSC.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > ou par une motion

File: locale/fr/about/governance.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourront être affiliés au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le 
+chamngement d'employeur d'un membre du TSC, créé une situation dans
+laquelle plus d'1/3 des membres du TSC partagent le même employeur,
+La situation devra immédiatement être corrigée par la démission ou le
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > la situation

File: locale/fr/about/governance.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+retrait d'un ou plusieurs membres du TSC affiliés avec le ou les 
+employeur-s sur-représenté-s.
+
+## Réunions du TSC
+
+Le TSC se réunit de manière hebdomadaire sur Google Hangout On Air.
+La réunion est animée par un modérateur désigné et approuvé apr le TSC.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > approuvé par le TSC

File: locale/fr/about/governance.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+ou sont des modifications de gouvernance, de politique de contribution, de
+participation au TSC, ou concernant le processus de release.
+
+L'intention du programme n'est pas d'approuver ou relire tous les 
+correctifs. Ceci devra être fait de manière continuelle sur GitHub
+et géré par le groupe des Collaborateurs.
+
+N'importe quel membre de la communauté ou contributeur peut demander
+à ajouter un objet au programme de la prochaine réunion en ouvrant un
+ticket sur GitHub. N'importe quel Collaborator, membre du TSC ou le
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > quel Collaborateur

File: locale/fr/about/governance.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+modérateur peut ajouter des objets au programme en ajoute l'étiquette
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > en ajoutant

File: locale/fr/about/governance.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+objet du programme et de les envoyer sous forme de pull request après
+chaque réunion.
+
+## Processus de Recherche de Consensus
+
+Le TSC suit un modèle décisionaire de
+[Recherche de Consensus](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+(en anglais).
+
+Quand on objet du programme a atteint un consensus, le modérateur
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un objet

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Thank you for that!

Not much to change, mostly small typohs.

I still need to check for translation issues. I'll do it later in another review.

**Comment by Tiriel** [comment]
Thanks! Typos fixed, I'll wait for your other review.

By the way, how can one join the website team and/or one of the localization teams? I'm currently investing myself hard, and I volunteered already in multiples issues (here, in the nodejs-fr repo, and in the CommComm repo and sent application to the team)

**Comment by fhemberger** [comment]
@Tiriel Usually, as you contribute, you get added to the Website WG's contributors list (this is a manual process at the moment done every couple of months), thus making you a full website team member.

Somehow you slipped through (maybe because you were already an org member), so I just added you manually. Sorry for that! As for nodejs-fr, @snostorm, can you please add him as well?

**Comment by Tiriel** [comment]
@fhemberger No problem at all, and thanks a lot for adding!
This is awesome! (And that must be like the tenth time I said it on one of the nodejs repos, but I stand by it).

File: locale/fr/about/governance.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+
+## Siéger au TSC
+
+Les sièges au TSC ne sont pas limités dans le temps. Le TSC n'a pas
+de taille fixe. Cependant, la cible est entre 6 et 12 membres, afin
+d'assurer une couverture suffisante des différents domaines d'expertise,
+en conservant la possibilité de prendre des décisions de manière
+efficace.
+
+Il n'y a aucun prérequis ou qualifications nécessaires au delà de
```
Comment (cedced19) [inline locale/fr/about/governance.md]: au-delà

File: locale/fr/about/governance.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+
+* la Direction Technique
+* La gouvernance et la gestion du projet (y compris cette politique)
+* La politique de contribution
+* La gestion des dépôts GitHub
+* Les guides de Conduite
+* La maintenance de la liste des Collaborateurs
+
+Les invitations originelles à siéger au TSC ont été proposées
+à des contributeurs actifs qui avaient une experience significative
```
Comment (cedced19) [inline locale/fr/about/governance.md]: expérience

File: locale/fr/about/governance.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+_Note:_ Si vous faites des contributions significatives et n'avez pas encore
+obtenu les droits de commit, ouvrez un ticket ou contactez un membre du TSC
+directement et votre demande sera examinée lors de la prochaine réunion.
+
+Les modifications de contenu sur le dépôt nodejs/node sont validées de
+manière collaborative. N'importe qui possédant un compte GitHub peut
+proposer une modification par pull request et elle sera examinée par les
+Collaborateurs du projet. Toutes les pull request doivent être relues et acceptées
+par un Collaborateur ayant une expertise suffisante et qui soit capable
+d'assumer la résponsabilité du changement effectué. Dans le cas d'une pull
```
Comment (cedced19) [inline locale/fr/about/governance.md]: r**e**sponsabilité

File: locale/fr/about/governance.md [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+Un membre du TSC peut être retiré du TSC par démission volontaire, ou
+par une motion standard du TSC.
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourront être affiliés au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le 
+chamngement d'employeur d'un membre du TSC, créé une situation dans
```
Comment (cedced19) [inline locale/fr/about/governance.md]: changement

File: locale/fr/about/governance.md [hunk 16]
```diff
@@ -0,0 +1,147 @@
...
+quel objet de leur choix au programme au début de chaque réunion. Le
+modérateur et le TSC ne peuvent retirer ou poser de veto sur les objets.
+
+Le TSC peut inviter des personnes ou des représentants de certains projets
+à participer sans pouvoir de vote. Ces invitations sont actuellement:
+
+* Un représentant du projet [build](https://github.com/node-forward/build)
+  choisi par ce projet.
+
+Le moderateur est chargé de résumer la discussion concernant chaque
```
Comment (cedced19) [inline locale/fr/about/governance.md]: mod**é**rateur

File: locale/fr/about/governance.md [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+objet du programme et de les envoyer sous forme de pull request après
+chaque réunion.
+
+## Processus de Recherche de Consensus
+
+Le TSC suit un modèle décisionaire de
```
Comment (cedced19) [inline locale/fr/about/governance.md]: décisionnaire

File: locale/fr/about/governance.md [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+[Recherche de Consensus](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+(en anglais).
+
+Quand un objet du programme a atteint un consensus, le modérateur
+demande une dernière fois "Quelqu'un a-t'il une objection?" pour valider
+le consensus.
+
+Si un objet du programme n'atteint pas de consensus, un membre du TSC
+peut soit appeler à un vote de cloture, ou à un vote pour reporter
```
Comment (cedced19) [inline locale/fr/about/governance.md]: cl**ô**ture

**Review by cedced19 — CHANGES_REQUESTED** [review]

**Comment by Tiriel** [comment]
Fixed ;)

File: locale/fr/about/governance.md [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+
+## Comité de Pilotage Technique
+
+Le projet est co-dirigé par un Comité de Pilotage Technique
+(Technical Steering Committee - TSC) qui est responsable de
+la gouvernance de haut-niveau du projet.
+
+Le TSC a toute autorité sur ce projet, y compris:
+
+* la Direction Technique
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > La direction technique

File: locale/fr/about/governance.md [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+* La gouvernance et la gestion du projet (y compris cette politique)
+* La politique de contribution
+* La gestion des dépôts GitHub
+* Les guides de Conduite
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > conduite

No need for uppercase I think.

File: locale/fr/about/governance.md [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+* La maintenance de la liste des Collaborateurs
+
+Les invitations originelles à siéger au TSC ont été proposées
+à des contributeurs actifs qui avaient une expérience significative
+en gestion de projet. La participation à ce comité est susceptible
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > significative avec la gestion du projet.

File: locale/fr/about/governance.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
...
+Vous pouvez trouver la liste des membres actuels du TSC sur le projet
+[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
+
+## Collaborateurs
+
+Le dépôt GitHub [nodejs/node](https://github.com/nodejs/node) est 
+maintenu par le TSC et un groupe de Collaborateurs additionnels
+qui sont ajoutés par le TSC de manière continue.
+
+Les personnes proposant des contributions significatives sont faits
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > sont faites Collaborateurs

File: locale/fr/about/governance.md [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+Collaborateurs et se voient accorder les droits d'écriture sur le projet.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > les droits de commit sur

Or, update the `droits de commit` in the next paragraph by `droits en écriture` to keep the same term for both.

File: locale/fr/about/governance.md [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+Ces personnes sont identifiées par le TSC et leur ajout aux Collaborateurs
+est discuté lors des réunions hebdomadaires du TSC
+
+_Note:_ Si vous faites des contributions significatives et n'avez pas encore
+obtenu les droits de commit, ouvrez un ticket ou contactez un membre du TSC
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > les droits de commit

See previous note.

File: locale/fr/about/governance.md [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+directement et votre demande sera examinée lors de la prochaine réunion.
+
+Les modifications de contenu sur le dépôt nodejs/node sont validées de
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > Les modifications du contenu du dépôt nodejs/node sont faites de

File: locale/fr/about/governance.md [hunk 26]
```diff
@@ -0,0 +1,147 @@
...
+manière collaborative. N'importe qui possédant un compte GitHub peut
+proposer une modification par pull request et elle sera examinée par les
+Collaborateurs du projet. Toutes les pull request doivent être relues et acceptées
+par un Collaborateur ayant une expertise suffisante et qui soit capable
+d'assumer la responsabilité du changement effectué. Dans le cas d'une pull
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > d'assumer l'entière responsabilité du changement

File: locale/fr/about/governance.md [hunk 27]
```diff
@@ -0,0 +1,147 @@
...
+request proposée par un Collaborateur, un autre Collaborateur est requis
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un Collaborateur additionnel est requis

File: locale/fr/about/governance.md [hunk 28]
```diff
@@ -0,0 +1,147 @@
...
+* Les guides de Conduite
+* La maintenance de la liste des Collaborateurs
+
+Les invitations originelles à siéger au TSC ont été proposées
+à des contributeurs actifs qui avaient une expérience significative
+en gestion de projet. La participation à ce comité est susceptible
+d'évoluer dans le temps en rapport avec les besoins du projet.
+
+Vous pouvez trouver la liste des membres actuels du TSC sur le projet
+[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > voir le README.md du projet.

File: locale/fr/about/governance.md [hunk 29]
```diff
@@ -0,0 +1,147 @@
...
+bas pour plus de détails sur le modèle de consensus utilisé.
+
+Les Collaborateurs peuvent choisir de faire remonter au TSC pour
+discussion des modifications significatives ou sujettes à controverse,
+ou des modifications n'ayant pas obtenu de consensus, en leur assignant
+l'étiquette ***tsc-agenda*** sur une pull request ou un ticket. Le
+TSC servira alors d'arbitre final lorsque requis.
+
+Pour la liste des Collaborateurs actuels, voir le projet 
+[README.md](https://github.com/nodejs/node/blob/master/README.md#current-project-team-members).
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > voir le README.md du projet.

File: locale/fr/about/governance.md [hunk 30]
```diff
@@ -0,0 +1,147 @@
...
+
+Un guide des Collaborateurs est proposé sur
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > est maintenu sur

File: locale/fr/about/governance.md [hunk 31]
```diff
@@ -0,0 +1,147 @@
...
+Un guide des Collaborateurs est proposé sur 
+[COLLABORATOR_GUIDE.md](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md).
+
+## Siéger au TSC
+
+Les sièges au TSC ne sont pas limités dans le temps. Le TSC n'a pas
+de taille fixe. Cependant, la cible est entre 6 et 12 membres, afin
+d'assurer une couverture suffisante des différents domaines d'expertise,
+en conservant la possibilité de prendre des décisions de manière
+efficace.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: The sentence seems odd in comparaison with the source material, here's what I propose : 

> Cependant, l'objectif recherché est entre 6 et 12 personnes, afin d'assurer une couverture adéquate des différents domaines d'expertise, en balance avec une prise de décision efficace.

File: locale/fr/about/governance.md [hunk 32]
```diff
@@ -0,0 +1,147 @@
...
+
+Un membre du TSC peut être retiré du TSC par démission volontaire, ou
+par une motion standard du TSC.
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourront être affiliés au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: In french,  [1/3 is singular and should be written in full letters](http://www.aidenet.eu/grammaire04k.htm), not sure about english.

> Pas plus d'un tiers des membres du TSC ne pourra être affilié au même employeur.

File: locale/fr/about/governance.md [hunk 33]
```diff
@@ -0,0 +1,147 @@
...
+par une motion standard du TSC.
+
+Les changements de modalités de participation au TSC devront être mises
+au programme et peuvent être ajoutés comme n'importe quel autre
+objet de programme (voir "Réunions du TSC" plus bas).
+
+Pas plus d'1/3 des membres du TSC ne pourront être affiliés au même
+employeur. Si le retrait ou la démission d'un membre du TSC, ou le 
+changement d'employeur d'un membre du TSC, créé une situation dans
+laquelle plus d'1/3 des membres du TSC partagent le même employeur,
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > plus d'un tiers des membres du TSC partage

File: locale/fr/about/governance.md [hunk 34]
```diff
@@ -0,0 +1,147 @@
...
+la situation devra immédiatement être corrigée par la démission ou le
+retrait d'un ou plusieurs membres du TSC affiliés avec le ou les 
+employeur-s sur-représenté-s.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: Not sure about the `-s` here. I'd use the plural form only, like:

> le ou les employeurs sur représentés.

File: locale/fr/about/governance.md [hunk 35]
```diff
@@ -0,0 +1,147 @@
...
+Le TSC se réunit de manière hebdomadaire sur Google Hangout On Air.
+La réunion est animée par un modérateur désigné et approuvé par le TSC.
+Chaque réunion est publiée sur Youtube.
+
+Sont ajoutés au programme les objets qui sont considérés contentieux
+ou sont des modifications de gouvernance, de politique de contribution, de
+participation au TSC, ou concernant le processus de release.
+
+L'intention du programme n'est pas d'approuver ou relire tous les 
+correctifs. Ceci devra être fait de manière continuelle sur GitHub
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > de manière continue sur GitHub

File: locale/fr/about/governance.md [hunk 36]
```diff
@@ -0,0 +1,147 @@
...
+et géré par le groupe des Collaborateurs.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > géré par l'ensemble des Collaborateurs.

File: locale/fr/about/governance.md [hunk 37]
```diff
@@ -0,0 +1,147 @@
...
+N'importe quel membre de la communauté ou contributeur peut demander
+à ajouter un objet au programme de la prochaine réunion en ouvrant un
+ticket sur GitHub. N'importe quel Collaborateur, membre du TSC ou le
+modérateur peut ajouter des objets au programme en ajoutant l'étiquette
+***tsc-agenda*** au ticket.
+
+Avant chaque réunion du TSC, le modérateur partagera le Programme
+avec les membres du TSC. Les membres du TSC pourront ajouter n'importe
+quel objet de leur choix au programme au début de chaque réunion. Le
+modérateur et le TSC ne peuvent retirer ou poser de veto sur les objets.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > Ni le modérateur ni le TSC ne peuvent

File: locale/fr/about/governance.md [hunk 38]
```diff
@@ -0,0 +1,147 @@
...
+
+Le TSC peut inviter des personnes ou des représentants de certains projets
+à participer sans pouvoir de vote. Ces invitations sont actuellement:
+
+* Un représentant du projet [build](https://github.com/node-forward/build)
+  choisi par ce projet.
+
+Le modérateur est chargé de résumer la discussion concernant chaque
+objet du programme et de les envoyer sous forme de pull request après
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > de l'envoyer

File: locale/fr/about/governance.md [hunk 39]
```diff
@@ -0,0 +1,147 @@
...
+(en anglais).
+
+Quand un objet du programme a atteint un consensus, le modérateur
+demande une dernière fois "Quelqu'un a-t'il une objection?" pour valider
+le consensus.
+
+Si un objet du programme n'atteint pas de consensus, un membre du TSC
+peut soit appeler à un vote de clôture, ou à un vote pour reporter 
+l'objet à la prochaine réunion. L'appel au vote doit être approuvé par
+la majorité du TSC ou la discussion continue. Seule la majorité simple
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > la majorité du TSC, sinon la discussion continue.

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Thank you for the update, I've now reviewed the meaning of it all.

Your translation is really good, your terms are really well-found and the meaning stays mostly in par with the source. That's a great work of translation.

I've just added some nitpicks that I found diverged from the source.

**Comment by Tiriel** [comment]
Thanks! I made changes as requested, I just slightly adapted one phrase to make it sound better.

File: locale/fr/about/governance.md [hunk 40]
```diff
@@ -12,43 +12,43 @@ la gouvernance de haut-niveau du projet.
 
 Le TSC a toute autorité sur ce projet, y compris:
 
-* la Direction Technique
+* la direction technique
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: missing that first `La` uppercase 🤓

File: locale/fr/about/governance.md [hunk 41]
```diff
@@ -12,43 +12,43 @@ la gouvernance de haut-niveau du projet.
...
 Les invitations originelles à siéger au TSC ont été proposées
 à des contributeurs actifs qui avaient une expérience significative
-en gestion de projet. La participation à ce comité est susceptible
+avec la gestion du projet. La participation à ce comité est susceptible
 d'évoluer dans le temps en rapport avec les besoins du projet.
 
-Vous pouvez trouver la liste des membres actuels du TSC sur le projet
-[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
+Pour trouver la liste des membres actuels du TSC, voir le [README.md]
+(https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee) du projet).
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: I think you missed that last `)`

File: locale/fr/about/governance.md [hunk 42]
```diff
@@ -60,19 +60,18 @@ ou des modifications n'ayant pas obtenu de consensus, en leur assignant
...
 ## Siéger au TSC
 
 Les sièges au TSC ne sont pas limités dans le temps. Le TSC n'a pas
-de taille fixe. Cependant, la cible est entre 6 et 12 membres, afin
-d'assurer une couverture suffisante des différents domaines d'expertise,
-en conservant la possibilité de prendre des décisions de manière
-efficace.
+de taille fixe. Cependant, l'objectif recherché est entre 6 et 12 personnes, 
+afin d'assurer une couverture adéquate des différents domaines d'expertise, 
+tout en conservant une capacité de prise de décision efficace.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: :+1:

File: locale/fr/about/governance.md [hunk 43]
```diff
@@ -106,8 +105,8 @@ ou sont des modifications de gouvernance, de politique de contribution, de
 participation au TSC, ou concernant le processus de release.
 
 L'intention du programme n'est pas d'approuver ou relire tous les 
-correctifs. Ceci devra être fait de manière continuelle sur GitHub
-et géré par le groupe des Collaborateurs.
+correctifs. Ceci devra être fait de manière continue sur GitHub
+géré par le groupe des Collaborateurs.
```
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: This sentence sounds weird to me now.

Why removing the `et` part and not applying my suggestion of `par l'ensemble des Collaborateurs`?

It's more in par with the source material I think.

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Small review.

File: locale/fr/about/governance.md (hunk 43 above)
Comment (Tiriel) [reply locale/fr/about/governance.md]: Oh my. Sorry, I hadn't seen the last part of your suggestion. Correcting ASAP

**Comment by Tiriel** [comment]
Done! I hope I didn't forget anything this time :D

**Review by yoannmoinet — APPROVED** [review]
Looks good to me!

Thank you very much for following through it all !

Just need the approval from @cedced19 now, to ensure his review updates are ok.

File: locale/fr/about/governance.md [hunk 44]
```diff
@@ -0,0 +1,146 @@
...
+title: Gouvernance du Projet
+layout: about.hbs
+---
+# Gouvernance du Projet
+
+## Comité de Pilotage Technique
+
+Le projet est co-dirigé par un Comité de Pilotage Technique
+(Technical Steering Committee - TSC) qui est responsable de
+la gouvernance de haut-niveau du projet.
```
Comment (clakech) [inline locale/fr/about/governance.md:11]: 'de haut-niveau' ne me semble pas naturel, peut être le supprimer ?

File: locale/fr/about/governance.md [hunk 45]
```diff
@@ -0,0 +1,146 @@
...
+laquelle plus d'1/3 des membres du TSC partage le même employeur,
+la situation devra immédiatement être corrigée par la démission ou le
+retrait d'un ou plusieurs membres du TSC affiliés avec le ou les 
+employeurs sur-représentés.
+
+## Réunions du TSC
+
+Le TSC se réunit de manière hebdomadaire sur Google Hangout On Air.
+La réunion est animée par un modérateur désigné et approuvé par le TSC.
+Chaque réunion est publiée sur Youtube.
```
Comment (clakech) [inline locale/fr/about/governance.md:101]: ajouter un lien ici ?

File: locale/fr/about/governance.md [hunk 46]
```diff
@@ -0,0 +1,146 @@
...
+
+Sont ajoutés au programme les objets qui sont considérés contentieux
```
Comment (clakech) [inline locale/fr/about/governance.md:103]: objets => sujets ?

File: locale/fr/about/governance.md [hunk 47]
```diff
@@ -0,0 +1,146 @@
...
+
+Avant chaque réunion du TSC, le modérateur partagera le Programme
+avec les membres du TSC. Les membres du TSC pourront ajouter n'importe
+quel objet de leur choix au programme au début de chaque réunion. Ni le
+modérateur ni le TSC ne peuvent retirer ou poser de veto sur les objets.
+
+Le TSC peut inviter des personnes ou des représentants de certains projets
+à participer sans pouvoir de vote. Ces invitations sont actuellement:
+
+* Un représentant du projet [build](https://github.com/node-forward/build)
```
Comment (clakech) [inline locale/fr/about/governance.md:125]: this link should be nodejs/build nope ?

**Review by clakech — APPROVED** [review]
great thanks !

**Comment by clakech** [comment]
@fhemberger ok for me, can we remove the on hold tag ?

**Comment by fhemberger** [comment]
@clakech Waiting for final feedback from @cedced19, then it can be merged.

File: locale/fr/about/governance.md (hunk 44 above)
Comment (Tiriel) [reply locale/fr/about/governance.md:11]: C'était pour essayer de coller au plus près de l'original. 'gouvernance générale'?

File: locale/fr/about/governance.md (hunk 45 above)
Comment (Tiriel) [reply locale/fr/about/governance.md:101]: Ca peut-être une bonne idée oui. Mais dans ce cas, peut-être voir pour l'ajouter dans toutes les trad et surtout sur le site `en` non?

File: locale/fr/about/governance.md (hunk 47 above)
Comment (Tiriel) [reply locale/fr/about/governance.md:125]: Didn't touch the link. @nodejs/website can someone answer? Do we need to change the link? On all pages?

Comment (lpinca) [reply locale/fr/about/governance.md:125]: It redirects to https://github.com/nodejs/build but yes it should be updated. It is probably better to do it in a dedicated PR though.

**Comment by Tiriel** [comment]
Another ping to @cedced19 just in case, but since all his suggested changes have been adressed, maybe this can be merged tomorrow in any case?

**Comment by fhemberger** [comment]
I'm merging this now. The changes are addressed, details can be added in an additional PR later.

**Pull request merged**